Gulpfile.ts

```diff
@@ -137,13 +137,16 @@ const es2017LibrarySourceMap = es2017LibrarySource.map(function(source) {
 });
 
 const hostsLibrarySources = ["dom.generated.d.ts", "webworker.importscripts.d.ts", "scripthost.d.ts"];
+const es2015HostLibrarySources = ["dom.iterable.d.ts", "streams.d.ts",  "fetch.d.ts"]

```

Comment:
i do not think this should be added to the es6 library. it is not related to the ES6 spec, it is an HTML standard.